oarepo_communities/ui/communities_components/templates/RecordCommunities.jinja

```diff
@@ -0,0 +1,56 @@
+{# def communities, imagePlaceholder='/static/images/square-placeholder.png', displayedCommunities=3 #}
+<section class="ui segment record-communities" aria-label='{{ _("Communities") }}'>
+    <h2 class="ui small header detail-sidebar-header">{{ _("Communities") }}</h2>
+    <div class="ui unstackable list">
+        {% for community in communities[0:displayedCommunities] %}
```

Comment:
I won't do a hard limit here. Num of communities would be mostly very low, and if there vere just 4, this would cause confusion about why 4th is not showing up